web/src/types/index.ts

```diff
@@ -36,6 +83,11 @@ export type KotsParams = {
   slug: string;
 };
 
+export type DashboardActionLink = {
```

Comment:
some of these names are not great but i figure we'll have a proper chat about kots domain language once we get all of the stuff typed out